Algorithm.Python/QuandlImporterAlgorithm.py

```diff
@@ -0,0 +1,61 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Indicators")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Indicators import *
+from QuantConnect.Data.Custom import *
+
+
+class QuandlImporterAlgorithm(QCAlgorithm):
+    '''QuantConnect University: Generic Quandl Data Importer
+    Using the underlying dynamic data class "Quandl" we take care of the data 
+    importing and definition for you. Simply point QuantConnect to the Quandl Short Code.
+     
+    The Quandl object has properties which match the spreadsheet headers.
+    If you have multiple quandl streams look at data.Symbol to distinguish them.'''
+    
+    __sma = None
+    __quandlCode = "YAHOO/INDEX_SPY"
+
+    def Initialize(self):
+        '''Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.'''
+
+        today = date.today()
+
+        self.SetStartDate(2013,1,1)                          #Set Start Date
+        self.SetEndDate(today.year, today.month, today.day)  #Set End Date
+        self.SetCash(25000)                                  #Set Strategy Cash
+        self.AddData[Quandl](self.__quandlCode, Resolution.Daily)
```

Comment:
So far I could only make it work when the type is defined in C#. When I wrote a class in Python (see `QuandlFuturesAlgorithm` for example), it doesn't work.